cli/smartcontract/smart_contract.go

```diff
@@ -214,16 +218,42 @@ func NewCommands() []cli.Command {
     * 'string\:string' is a string with a value of 'string:string'
     * '03b209fd4f53a7170ea4444e0cb0a6bb6a53c2bd016926989cf85f9b0fba17a70c' is a
       key with a value of '03b209fd4f53a7170ea4444e0cb0a6bb6a53c2bd016926989cf85f9b0fba17a70c'
+
+   Cosigners represent a set of Uint160 hashes with witness scopes and are used
+   to verify hashes in System.Runtime.CheckWitness syscall. To specify cosigners
+   use [cosigner]:[scope] syntax were
```

Comment:
`cosigner[:scope]`, `[]` is usually used for optional things. And you should mention somewhere here that scopes only with neo-go RPC server.